cdap-docs/developers-manual/source/data-exploration/streams.rst

```diff
@@ -202,6 +202,8 @@ When creating your queries, keep these limitations in mind:
 - When addressing your streams in queries, you need to prefix the stream name with
   ``stream_``. For example, if your Stream is named ``Purchases``, then the corresponding table
   name is ``stream_purchases``. Note that the table name is all lower-case, regardless of how it was defined.
+- If your Stream name contains a ‘.’ or a ‘-‘, those characters will be converted to ‘_’ for the Hive table name.
```

Comment:
No, those are -‘-, not -'-. (I know, hard to tell the difference)... Compare line 205 with line 118 in the previous file...they are not the same. You could also change to words instead of symbols (such as "contains a period or a hyphen, those characters will be converted to an underscore for the Hive table name"...)